polymer/slambook/lib/tute_slambookform.html

```diff
@@ -0,0 +1,94 @@
+<!--
+  Copyright (c) 2012, the Dart project authors.  Please see the AUTHORS file
+  for details. All rights reserved. Use of this source code is governed by a
+  BSD-style license that can be found in the LICENSE file.
+-->
+
+<dom-module id="tute-slambookform" extends="form">
+  <template>
+    <style>
+      #slambookform {
+          width: 500px;
+       }
+
+       .entry {
+           float: left;
+           padding: 3px 3px 3px 3px;
+           width: 100%;
+       }
+
+       .entry label {
+           display: inline-block;
+           width: 150px;
+           text-align: right;
+           vertical-align: middle;
+       }
+
+       .submitarea {
+           padding: 10px 10px 10px 10px;
+           margin: 10px 10px 10px 10px;
+           border-style: solid;
+           border-width: 1px;
+           border-color: SlateBlue;
+           clear: left;
+       }
+
+       .responsetext {
+           padding: 5px 10px 10px 10px;
+           margin: 10px 10px 10px 10px;
+           clear: left;
+           background-color: WhiteSmoke;
+           width: 90%;
+           height: 70px;
+           resize: none;
+       }
+     </style>
+     <div id="slambookform">
+       <div class="entry">
+         <label>First name?</label>
+         <input type="text" value="{{myData.firstName::change}}">
+       </div>
+       <div class="entry">
+         <label>Favorite quote?</label>
+         <input type="text" value="{{myData.favoriteQuote::change}}">
+       </div>
+       <div class="entry">
+         <label>Favorite color?</label>
+         <input type="color" value="{{myData.favoriteColor::change}}">
+       </div>
+       <div class="entry">
+         <label>Birthday?</label>
+         <input type="date" value="{{myData.birthday::change}}">
+       </div>
+       <div class="entry">
+         <label>Cat, dog, or iguana?</label>
+         <input on-change="petChanged" name="pet" type="radio" value="cat" checked="{{myData.pet}}">Cat
```

Comment:
The checked binding here doesn't really do anything, you can just have the checked attribute (its value does not matter):

```html
<input on-change="petChanged" name="pet" type="radio" value="cat" checked>Cat
```